.github/workflows/tools/dispatch-build.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+__requires__ = ["click ~= 7.0", "PyGithub == 1.*"]
+import os
+import subprocess
+import click
+from github import Github
+
+REPO = "datalad/git-annex"
+ALL_OS_TYPES = ("macos", "ubuntu", "windows")
+
+
+class BuildDispatcher:
+    def __init__(self, token):
+        self.gh = Github(token)
+        self.repo = self.gh.get_repo(REPO)
+
+    def get_os_workflows(self, ostypes):
+        return [self.repo.get_workflow(f"build-{o}.yaml") for o in ostypes]
+
+    def build_pr(self, pr, ostypes=ALL_OS_TYPES, workflow_ref="master"):
+        for w in self.get_os_workflows(ostypes):
+            w.create_dispatch(ref=workflow_ref, inputs={"pr": str(pr)})
+
+    def build_commitish(self, commitish, ostypes=ALL_OS_TYPES, workflow_ref="master"):
+        for w in self.get_os_workflows(ostypes):
+            w.create_dispatch(ref=workflow_ref, inputs={"commitish": commitish})
+
+    def build_latest(self, ostypes=ALL_OS_TYPES, workflow_ref="master"):
+        for w in self.get_os_workflows(ostypes):
+            w.create_dispatch(ref=workflow_ref, inputs={})
+
+
+@click.command()
+@click.option(
+    "--os",
+    "ostypes",
+    type=click.Choice(["macos", "ubuntu", "windows"]),
```

Comment:
```suggestion
    type=click.Choice(ALL_OS_TYPES),
```
?

Reply:
Applied manually (because "Outdated suggestions cannot be applied.")